ftdetect/mako.vim

```diff
@@ -1 +1,11 @@
+if !exists("g:mako_detect_lang_from_ext")
+  let g:mako_detect_lang_from_ext = 1
+endif
+if g:mako_detect_lang_from_ext
+  au BufNewFile *.*.mako   execute "do BufNewFile " . expand("<afile>:r") | let b:mako_outer_lang = &filetype
```

Comment:
OK, so if i understand what's happening here the file does everything that would be associated with a new file of the outer_lang type (or below, with reading that type)... then sets the outer_lang to the detected filetype.

My concern is that there may be things other than filetype detection associated with the "BufRead" event. There's a lot of odd plugins out there! Perhaps this should be limited to `execute "do BufRead filetypedetect "...`?